pages/vi/profiles/andykais.md

```diff
@@ -0,0 +1,22 @@
+# Andrew Kaiser
+
+![iceland glaciers photo](https://farm8.staticflickr.com/7083/26781928475_7078d3af66_k_d.jpg)
+glaciers in iceland!
+
+```html
```

Comment:
uses of tags seems good and you tried to use different way of displaying. PR looks good just not sure about displaying info as code block :) will let others decide on it.

Reply:
ok, happy to change to a regular bullet list if others decide its too wierd

Reply:
Wise use of the markdown elements looks good, doesn't seem weird to me :)